openedx_learning/apps/authoring/collections/models.py

```diff
@@ -0,0 +1,60 @@
+"""
+Core models for Collections
+"""
+from __future__ import annotations
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from ....lib.fields import case_insensitive_char_field
+from ..publishing.models import LearningPackage
+
+
+class Collection(models.Model):
+    """
+    Represents a collection of library components
+    """
+
+    id = models.AutoField(primary_key=True)
+    learning_package = models.ForeignKey(LearningPackage, on_delete=models.CASCADE)
```

Comment:
Please add a comment or help text saying that every collection is scoped to a LearningPackage / owned by a LearningPackage.

Reply:
Sure! Done: 28a96f932b33fbc1a48f0d52098958c4da907adf